airbyte-integrations/connectors/source-facebook-marketing/setup.py

```diff
@@ -25,4 +25,9 @@
     extras_require={
         "tests": TEST_REQUIREMENTS,
     },
+    entry_points={
```

Comment:
what's the best way to test this?

Reply:
For now, the easiest way is this:
* Go to `airbyte-lib`
* Get a shell: `poetry shell`
* Go to the connector directory
* Install into current venv: `pip install .`
* Open python REPL: `python`
* Use the connector with `use_local_install=True`:
```python
import airbyte_lib as ab

c = ab.get_connector("source-facebook-marketing", config={... config object ...}, use_local_install=True)

# test things

c.check()

c.get_available_streams()

next(c.read_stream("<stream name>"))
```